LoginDemo/LearningSelenium/PageModel/Login.cs

```diff
@@ -0,0 +1,84 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace LearningSelenium.PageModel
+{
+    public class Login
+    {
+        WebDriver driver;
+        protected By UserName = By.XPath("//input[contains(@name,'username')]");
+        protected By Password = By.XPath("//input[contains(@name,'password')]");
```

Comment:
Usually elements have private access modifier because we work with elements using the methods from Page Objects. Of course, there may be some exceptions.